lib/storage/file.js

```diff
@@ -902,14 +902,14 @@ File.prototype.getMetadata = function(callback) {
  *     members.
  *
  * @param {object} options - Configuration object.
- * @param {object} options.expiration - Timestamp (seconds since epoch) when
- *     this policy will expire.
  * @param {array|array[]=} options.equals - Array of request parameters and
  *     their expected value (e.g. [['$<field>', '<value>']]). Values are
  *     translated into equality constraints in the conditions field of the
  *     policy document (e.g. ['eq', '$<field>', '<value>']). If only one
  *     equality condition is to be specified, options.equals can be a one-
  *     dimensional array (e.g. ['$<field>', '<value>']).
+ * @param {object} options.expires - Timestamp (seconds since epoch) when this
```

Comment:
My stance on breaking our API is the same as always. Breaking changes on pre-1.0 minor versions is why we bump the minor. If we are making things better, then I don't think we should worry about backwards compatibility... at least not for this kind of thing. If it was like we changed "createBucket(name, options)" to "createBucket(options)", then we should probably legacy support that for 1-2 next releases.

But, this is basically an "API fix" - an inconsistency between our docs and code, and between what similar methods accept as input. We need to fix it, and I don't think we need to backwards support it.